.builds/openbsd.yml

```diff
@@ -16,26 +16,28 @@ packages:
 sources:
 - https://github.com/neovim/neovim
 
+environment:
+  SOURCEHUT: 1
+  LC_CTYPE: en_US.UTF-8
+  CMAKE_EXTRA_FLAGS: -DTRAVIS_CI_BUILD=ON
+
 tasks:
-- build: |
+- build-deps: |
     export AUTOCONF_VERSION=2.69
     export AUTOMAKE_VERSION=1.15
-    cd neovim
-    mkdir .deps
-    cd .deps
+    mkdir neovim/.deps
+    cd neovim/.deps
     cmake -G Ninja ../third-party/
     cmake --build . --config Debug
-    cd ..
-    mkdir build
-    cd build
-    cmake -G Ninja -DMIN_LOG_LEVEL=3 ..
```

Comment:
`MIN_LOG_LEVEL=3` is intentional and should be used in all CI jobs (faster (avoids disk) and less noisy for the `NVIM_LOG_FILE` tail). 

We might change to `MIN_LOG_LEVEL=2` after we sort out the warning-level messages. https://github.com/neovim/neovim/pull/8917